src/screens/claimables/ClaimClaimablePanel.tsx

```diff
@@ -0,0 +1,232 @@
+import React from 'react';
+import { AccentColorProvider, Bleed, Box, Inline, Text, TextShadow, globalColors, useColorMode } from '@/design-system';
+import * as i18n from '@/languages';
+import { ListHeader, Panel, TapToDismiss, controlPanelStyles } from '@/components/SmoothPager/ListPanel';
+import { ChainNameDisplay } from '@/networks/types';
+import { useAccountSettings } from '@/hooks';
+import { safeAreaInsetValues } from '@/utils';
+import { View } from 'react-native';
+import { IS_IOS } from '@/env';
+import { ButtonPressAnimation } from '@/components/animations';
+import { RouteProp, useRoute } from '@react-navigation/native';
+import { Claimable } from '@/resources/addys/claimables/types';
+import RainbowCoinIcon from '@/components/coin-icon/RainbowCoinIcon';
+import { useTheme } from '@/theme';
+import { FasterImageView } from '@candlefinance/faster-image';
+import { useClaimables } from '@/resources/addys/claimables/query';
+// import { ContextMenuButton } from '@/components/context-menu';
+
+type RouteParams = {
+  ClaimClaimablePanelParams: { uniqueId: string };
+};
```

Comment:
can you move these to `RootStackParamList`
https://github.com/rainbow-me/rainbow/blob/dd030ca15cc86207c1f5f78d611d20ed07891f40/src/navigation/types.ts#L19